x/rollup/keeper/msg_server.go

```diff
@@ -83,3 +86,70 @@ func (k *Keeper) InitiateWithdrawal(
 
 	return &types.MsgInitiateWithdrawalResponse{}, nil
 }
+
+func (k *Keeper) InitiateFeeWithdrawal(
+	goCtx context.Context,
+	_ *types.MsgInitiateFeeWithdrawal,
+) (*types.MsgInitiateFeeWithdrawalResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	// TODO: make minWithdrawalAmount and l1recipientAddr configurable once a genesis state is added
+	const (
+		minWithdrawalAmount   = 100
+		l1recipientAddr       = "0x63d93aC6FA6B4021527e967ac3Eb29F2B3E52B96"
+		feeWithdrawalGasLimit = 400_000
+	)
```

Comment:
Would it be easier to just make these params in the `MsgInitiateFeeWithdrawal`? Not necessary in this PR.

Reply:
These params will eventually be module specific once a genesis state is added instead of stored as constants.

However, they can't be added as a field to `MsgInitiateFeeWithdrawal` since any L2 user can permissionlessly trigger a fee withdrawal. If these were fields on `MsgInitiateFeeWithdrawal`, then they could maliciously forward the fees to a custom L1 recipient address instead of the predefined L1 address that the sequencer expects.